dev/tasks/gandiva-jars/build-java.sh

```diff
@@ -25,11 +25,11 @@ pushd java
   if [[ $OS_NAME == "linux" ]]; then
     SO_DEP=ldd
     GANDIVA_LIB="$CPP_BUILD_DIR"libgandiva_jni.so
-    WHITELIST=(linux-vdso libz librt libdl libpthread libstdc++ libm libgcc_s libc ld-linux-x86-64)
+    WHITELIST=(linux-vdso libz librt libdl libpthread libstdc++ libm libgcc_s libc ld-linux-x86-64 libssl libcrypto)
```

Comment:
can you make sure that the nightly build uses a static version of openssl as dependency. this whitelist is to make sure that there are no shared libraries added as dependencies when building the gandiva library in nightly

Reply:
Changes applied, the **ARROW_OPENSSL_USE_SHARED** variable will set if the library must be linked statically or dynamically.